htdocs/compta/bank/annuel.php

```diff
@@ -195,10 +195,10 @@
 for ($mois = 1; $mois < 13; $mois++) {
 	print '<tr class="oddeven">';
 	print "<td>".dol_print_date(dol_mktime(1, 1, 1, $mois, 1, 2000), "%B")."</td>";
-	for ($annee = $year_start; $annee <= $year_end; $annee++) {
-		$totsorties[$annee] = 0;
-		$totentrees[$annee] = 0;
 
+	$totsorties[$annee] = 0;
```

Comment:
You can't do $totsorties[$annee] = 0 before the loop because $annee is not define before the loop.
May be you can move the init of $totsorties before the for $month by doing a 
        for ($annee = $year_start; $annee <= $year_end; $annee++) {
$totsorties[$annee] = 0

But before the loop on month...